packages/shared/src/components/post/PostSourceInfo.tsx

```diff
@@ -83,8 +76,8 @@ function PostSourceInfo({
                 {source.handle}
               </a>
             </Link>
-            {showActionBtn && <Separator />}
-            {showActionBtn && source?.type !== SourceType.Squad && (
+            {isMobile && showActionBtn && <Separator />}
```

Comment:
Why do we need so many isMobile options here?
Maybe we can just wrap this whole thing in one big isMobile then?
but not sure it's even needed.